.github/workflows/updateIndex.yml

```diff
@@ -0,0 +1,53 @@
+
+name: Update Devices Index
+
+on:
+  pull_request:
+    types: [opened, edited]
+  issue_comment:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: Setup repo
+        run: |
+          npm install
+          npm run prerelease
+
+      - name: Update index
+        id: check_index
+        run: |
+          npm run config -- -i
+          indexPath="packages/config/config/devices/index.json"
+          changedFiles=$(git diff-index --name-only HEAD)
+          if [ "$changedFiles" = "$indexPath" ]; then
+            echo "Index has changed"
+            git add $indexPath
+            echo "::set-output name=PUSH::true"
+          fi
+      - name: Comment a pull_request
+        if: steps.check_index.outputs.PUSH == 'true' && github.event_name == 'pull_request'
+        uses: mb2dev/github-action-comment-pull-request@1.0.0
+        with:
+          message: "The index file isn't updated. Comment 'update index' to automatically update it"
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I discussed something similar with @chrisns lately. I'm already doing it with the automerge action, but I'm not really comfortable spreading the `GITHUB_TOKEN` to a bunch of open source actions 🤔
Commenting can be done with octokit, like here https://github.com/zwave-js/node-zwave-js/blob/master/.github/actions/toLogEntry/main.js

Reply:
Or even easier: https://github.com/actions/github-script#comment-on-an-issue

Reply:
Done